### ISSUE TITLE
Node v13.3.0 (Current)

### ISSUE THREAD

**Issue by Fleta** [nodejs/nodejs-ko#908]
https://nodejs.org/ko/blog/release/v13.3.0/

```
npm run scaffold https://raw.githubusercontent.com/nodejs/nodejs.org/master/locale/en/blog/release/v13.3.0.md 2019-12-03
```

**Comment by MJayJang** [comment]
번역 담당하겠습니다:)

**Comment by outsideris** [comment]
closed by #900
